src/data/combos/example1.yaml

```diff
@@ -0,0 +1,10 @@
+id: 1
```

Comment:
Perhaps more attributes (like for example a description) could be added here if these examples are used for testing. Else just close this comment.

Reply:
Ye. I set these examples just up to see if the Archiving and unarchiving Logic works. 
We could expand on the examples here — but in a separate issue